ocs_ci/ocs/ripsaw.py

```diff
@@ -42,9 +42,13 @@ def __init__(self, **kwargs):
         """
         self.args = kwargs
         self.repo = self.args.get(
-            "repo", "https://github.com/cloud-bulldozer/benchmark-operator"
+            # "repo", "https://github.com/cloud-bulldozer/benchmark-operator"
+            # This is emergency fix since the all benchmark-operator has changed
```

Comment:
@Avilir can you please raise an issue to fix this issue permanently so we track them?

Reply:
will do.